lib/validator.js

```diff
@@ -192,13 +192,13 @@ Specberus.prototype.getSotDSection = function () {
     if (!$startH2 || !$endH2) return null;
     var started = false;
     $startH2.parent().children().each(function () {
-        if ($startH2[0] === this[0]) {
+        if ($startH2[0] === this) {
             started = true;
             return;
         }
         if (!started) return;
-        if ($endH2[0] === this[0]) return false;
-        $div.append(this.clone());
+        if ($endH2[0] === this) return false;
+        $div.append($(this).clone());
     });
```

Comment:
Updating whacko from 0.13.x to 0.17.x made this piece of art break. I'm not sure why the changes above fixed the stuff, but it did. I couldn't find where this change became needed (along with whacko, a ton of dependencies evolved) so if you are aware of something, please share. I had a hard time fixing this...

In a nutshell, now `this[0]` is `undefined`, voilà.